### DISCUSSION TITLE
Introduce an agent splitting function

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5728] [Feature Requests]
This would allow parsing the agents that exist in long text fields

**Comment by maxpatiiuk** [comment]
Haven't given much though to this, but at first glance, this may be simpler than agent merging.
A few corner cases:
What if agent was used in a -to-one relationship? Which of the splitted records should be associated o the -to-one? Or should an agent group be created?

If so, this becomes way simpler - we can just have a tool that allows to separate separate people from the last name string into agent records, all of which are grouped by an Agent type "group"
